static/app/views/settings/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -292,6 +329,78 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
       </Fragment>
     );
   }
+
+  renderFeatures() {
+    const {configurations} = this.state;
+    const {organization} = this.props;
+    const hasIntegration = configurations ? configurations.length > 0 : false;
+    const endpoint = `/organizations/${organization.slug}/`;
+    const hasOrgWrite = organization.access.includes('org:write');
+
+    const forms: JsonFormObject[] = [
+      {
+        fields: [
+          {
+            name: 'githubPRBot',
+            type: 'boolean',
+            label: t('Enable Pull Request Bot'),
+            visible: ({features}) => features.has('pr-comment-bot'),
+            help: (
+              <Fragment>
+                {t(
+                  'Allow Sentry to comment on pull requests about issues impacting your app.'
+                )}
+              </Fragment>
+            ),
+            disabled: !hasIntegration || !hasOrgWrite,
+            disabledReason: (
+              <Fragment>
+                {t('You must have a GitHub integration to enable this feature.')}
+              </Fragment>
+            ),
+          },
+        ],
+      },
+    ];
+
+    const jsonFormSettings = {
+      features: new Set(organization.features),
+    };
+
+    return (
+      <Form
+        data-test-id="organization-settings"
+        apiMethod="PUT"
+        apiEndpoint={endpoint}
+        saveOnBlur
+        allowUndo
+        initialData={organization}
```

Comment:
i'm not sure this makes sense? Shouldn't this be something else

Reply:
if i remove it, it doesn't fetch the org option data correctly to populate the toggle